website/docs/commands/garnet-specific.md

```diff
@@ -148,7 +148,7 @@ Garnet provides support for ETags on raw strings. By using the ETag-related comm
 
 Compatibility with non-ETag commands and the behavior of data inserted with ETags are detailed at the end of this document.
 
-To initialize a key value pair with an ETag you can use either the SET command with the newly added "WITHETAG" optional flag, or you can take any existing Key value pair and call SETIFMATCH with the ETag argument as 0 (Any key value pair without an explicit ETag has an ETag of 0 implicitly). You can read more about setting an initial ETag via SET [here](../commands/raw-string#set)
+To initialize a key value pair with an ETag you can use either the SET command with the newly added "WITHETAG" optional flag, or you can take any existing Key value pair and call SETIFMATCH with the ETag argument as 0 (Any key value pair without an explicit ETag has an ETag of 0 implicitly). **You can read more about setting an initial ETag via SET [here](../commands/raw-string#set)**
```

Comment:
As discussed, expand out SET ... WITHETAG into its own section here. It is easy to miss it. OK to include in this PR even though this is unrelated.